packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -122,7 +122,9 @@ class CupertinoTextSelectionControls extends TextSelectionControls {
     final Widget handle;
 
     final Widget customPaint = CustomPaint(
-      painter: _CupertinoTextSelectionHandlePainter(CupertinoTheme.of(context).primaryColor),
+      painter: _CupertinoTextSelectionHandlePainter(
```

Comment:
I wonder if here we should add `primaryColor` as a fallback since there might be some users who were going under the previous assumption that `primaryColor` controls the selection handle color.

Reply:
Fair enough, but then I'd have to make the `selectionHandleColor` nullable in the `CupertinoTheme` and also remove it from the `_CupertinoTextThemeDefaults`. What do you think?

Reply:
Oh in that case the way we have it right now (with defaults) sounds good to me. All the tests are passing so this should be fine.